### ISSUE TITLE
Boat color is not visible on Bedrock

### ISSUE THREAD

**Issue by JuniorJPDJ** [GeyserMC/Geyser#237]
**Describe the bug**
Boats always have the same color (normal oak)

**To Reproduce**
1. Create non-oak boat on java edition
2. Look at boat from bedrock
3. Boat is visible as oak one

**Expected behavior**
Boat should be the same color/variant in both games

**Server version**
Paper 1.15.2

**Geyser version**
Latest inventory atm

**Bedrock version**
1.14.1.5 android arm

**Comment by DoctorMacc** [comment]
This is the same problem that beds had and skulls have - a different between a block state in Bedrock (or in this case a data value) and a namespaced ID in Java.

**Comment by RednedEpic** [comment]
> This is the same problem that beds had and skulls have - a different between a block state in Bedrock (or in this case a data value) and a namespaced ID in Java.

The boat variant as sent as part of the entity metadata. In the mounts branch, this problem is no longer a thing. Once issues are fixed in that branch and is merged into master, this will be closed.
